chain/jsonrpc/tests/rpc_query.rs

```diff
@@ -452,7 +452,13 @@ fn test_validators_ordered() {
 fn test_genesis_config() {
     test_with_client!(test_utils::NodeType::NonValidator, client, async move {
         let genesis_config = client.EXPERIMENTAL_genesis_config().await.unwrap();
-        assert_eq!(genesis_config["protocol_version"].as_u64().unwrap(), PROTOCOL_VERSION as u64);
+        #[cfg(not(feature = "nightly_protocol"))]
```

Comment:
@frol I somehow cannot get this to work. With the way it is done currently, when I run the test with `--features nightly_protocol,nightly_protocol_features` enabled this part still gets triggered. Do you know how to fix it?

Reply:
You had to propagate the feature from `neard`. I fixed it for you.

Reply:
I am a bit confused as why that is necessary. Is it because how tests are built?